src/schema/course/resolvers.ts

```diff
@@ -0,0 +1,49 @@
+import { QueryResolvers } from '../../types/types';
+import normalizeKeys from '../../utils/normalizeKeys';
+import { CourseParams } from './types';
```

Comment:
You can also use generated types `import {QueryCourseListArgs} from "../../types/types"`